app/packages/components/src/components/AdaptiveMenu/index.tsx

```diff
@@ -53,7 +59,9 @@ export default function AdaptiveMenu(props: AdaptiveMenuPropsType) {
     if (!containerElem) return;
     hideOverflowingNodes(containerElem, (_: number, lastVisibleItemId) => {
       const lastVisibleItem = itemsById[lastVisibleItemId];
-      const computedHidden = items.length - lastVisibleItem.index - 1;
+      const computedHidden = lastVisibleItem?.index
```

Comment:
Are we expecting the lastVisibleItem to be `undefined`? It looks this indicates some other bug with `lastVisibleItemId`

Reply:
Not sure about the entire call stack but the direct error is the lastItem being none so therefore not having an index property (see error screenshot). In my ide, it highlights this as well warning that the value may be undefined